.baseline/checkstyle/checkstyle.xml

```diff
@@ -134,8 +134,7 @@
             <message key="import.illegal" value="Must not import repackaged classes."/>
         </module>
         <module name="IllegalImport"> <!-- Java Coding Guidelines: Import the canonical package -->
-            <property name="id" value="BanShadedClasses"/>
-            <property name="illegalPkgs" value=".*\.(repackaged|shaded|thirdparty)"/>
+            <property name="illegalPkgs" value=".*\.(repackaged|thirdparty)"/>
```

Comment:
I'd prefer to keep this the same and use a different name for the package. That way the rule still applies to anything else that uses "shaded" in the name. What about `org.apache.iceberg.relocated.com.google.common...`?